File: Iniciante/1072 - Intervalo 2/JavaScript/1072.js
function main() {
  const input = require("fs").readFileSync(0, "utf8").trim().split(/\s+/);
  let index = 0;
  const quantityOfValues = Number(input[index++]);
  let inside = 0;
  let outside = 0;

  for (let i = 0; i < quantityOfValues; i++) {
    const value = Number(input[index++]);
    if (value >= 10 && value <= 20) {
      inside++;
    } else {
      outside++;
    }
  }

  console.log(`${inside} in`);
  console.log(`${outside} out`);
}

main();

File: Iniciante/1072 - Intervalo 2/Python/1072.py
def main():
    quantity_of_values = int(input())
    inside = 0
    outside = 0

    for _ in range(quantity_of_values):
        value = int(input())
        if 10 <= value <= 20:
            inside += 1
        else:
            outside += 1

    print(f"{inside} in")
    print(f"{outside} out")


if __name__ == "__main__":
    main()
